### PR TITLE
Release 3.10.1

### PR DESCRIPTION
**Pull request by forevermatt**
### Fixed
- Fix GitHub Actions integration to support `--tag-only` parameter
- Fix GitHub Actions integration to support `--aws-assume-role` parameter

### CODE REVIEW

**Comment by forevermatt** [comment]
It looks like #237 may also fit well in this release, so I'm waiting until we've reviewed (and possibly merged) that PR before releasing this update.

**Review by Schparky — APPROVED** [review]

**Review by Baggerone — APPROVED** [review]

**Pull request merged**
